packages/@lwc/engine-core/src/framework/vm.ts

```diff
@@ -190,7 +197,7 @@ export function rerenderVM(vm: VM) {
 export function connectRootElement(elm: any) {
     const vm = getAssociatedVM(elm);
 
-    startGlobalMeasure(GlobalMeasurementPhase.HYDRATE, vm);
+    logGlobalOperationStart(GlobalMeasurementPhase.HYDRATE, vm);
```

Comment:
wrap this with `if (profilerEnabled)`. see other uses of logOperationStart

Reply:
@tariqrafique Thank you! Fixed.